pkg/tree/simple/folder.go

```diff
@@ -127,10 +126,10 @@ func (f *Folder) listFolders(prefix string, maxDepth, curDepth int) []string {
 	for _, key := range sortedFolders(f.Folders) {
 		out = append(out, f.Folders[key].listFolders(prefix, maxDepth, curDepth+1)...)
 		// we want to also list the base folders, not just the leaves
-		out = append(out, path.Join(prefix, f.Folders[key].Name))
+		out = append(out, filepath.Join(prefix, f.Folders[key].Name))
 	}
 
-	sort.Strings(out)
```

Comment:
`sort.Strings` gives different results for `[foo, foo/bar, foo/bar/baz]` or `[foo,foo\bar,foo\bar\baz]`